examples/eks_argo/metaflow.tf

```diff
@@ -65,3 +65,8 @@ module "metaflow-metadata-service" {
 
   standard_tags = local.tags
 }
+
+variable "with_public_ip" {
+  type    = bool
+  default = true
```

Comment:
doesn't seem to be used anywhere in this PR?

Reply:
TF won't apply without it... existing bug.